### PR TITLE
chore: add api_shortname to repo-metadata.json

### PR DESCRIPTION
**Pull request by parthea**
Update .repo-metadata.json as required by go/library-data-integrity

### CODE REVIEW

File: .repo-metadata.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-  "release_level": "beta",
-  "language": "python",
-  "library_type": "INTEGRATION",
-  "repo": "googleapis/python-spanner-django",
-  "distribution_name": "django-google-spanner",
-  "requires_billing": true
+    "name": "django-google-spanner",
+    "name_pretty": "Cloud Spanner Django",
+    "product_documentation": "https://cloud.google.com/spanner/docs/",
+    "client_documentation": "https://cloud.google.com/python/docs/reference/django-google-spanner/latest",
```
Comment (parthea) [inline .repo-metadata.json:5]:
```suggestion
    "client_documentation": "https://googleapis.dev/python/django-google-spanner/latest",
```

File: .repo-metadata.json [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-  "library_type": "INTEGRATION",
-  "repo": "googleapis/python-spanner-django",
-  "distribution_name": "django-google-spanner",
-  "requires_billing": true
+    "name": "django-google-spanner",
+    "name_pretty": "Cloud Spanner Django",
+    "product_documentation": "https://cloud.google.com/spanner/docs/",
+    "client_documentation": "https://googleapis.dev/python/django-google-spanner/latest",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:190851%2B%20status:open",
+    "release_level": "preview",
```
Comment (vi3k6i5) [inline .repo-metadata.json:7]: @parthea : Please change `release_level` to `ga`.
`"release_level": "ga"`

**Review by vi3k6i5 — APPROVED** [review]
LGTM

File: .repo-metadata.json (hunk 2 above)
Comment (parthea) [inline .repo-metadata.json:7]:
```suggestion
    "release_level": "stable",
```

Comment (parthea) [reply .repo-metadata.json]: Good catch! I've updated the release level to stable.

**Pull request merged**